universum/lib/gravity.py

```diff
@@ -7,28 +9,30 @@
 
 
 class ModuleSettings:
-    def __init__(self, cls, main_settings):
+    def __init__(self, cls: Type['Module'], main_settings: 'ModuleSettings') -> None:
```

Comment:
`main_settings` is `universum.lib.module_arguments.ModuleNamespace`, please see https://github.com/Samsung/Universum/blob/042ed269b1138273217ca2185a903498b3bcfd38/tests/test_gravity.py#L187-L191

Reply:
will fix, my bad, all other main_settings seemed to point to ModuleSettings - which was pretty confusing. I think this annotation already makes it much clearer that ModuleSettings is a proxy to ModuleNamespace.

Reply:
All `main_settings` are instances of `ModuleNamespace`